util/grpc/Cargo.toml

```diff
@@ -2,8 +2,10 @@
 name = "mc-util-grpc"
 version = "4.0.2"
 authors = ["MobileCoin"]
-edition = "2021"
 description = "Runtime gRPC Utilities"
+edition = "2021"
+license = "GPL-3.0"
```

Comment:
There's potentially client code in here i think

Reply:
This is admin and health-check APIs that the servers use, so I don't think there's any client code in here.